receive_message_queue_job.py

```diff
@@ -11,8 +11,8 @@
 CONNECTION_STR = os.environ['SERVICE_BUS_CONNECTION_STR']
 QUEUE_NAME = os.environ["SERVICE_BUS_QUEUE_NAME"]
 CONNECTION_URL = os.environ["CONNECTION_URL"]
-MAX_MESSAGE_COUNT = os.getenv('MAX_MESSAGE_COUNT', default=1)
-MAX_WAIT_TIME = os.getenv('MAX_WAIT_TIME', default=5)
+MAX_MESSAGE_COUNT = os.getenv('MAX_MESSAGE_COUNT', default=10)
+MAX_WAIT_TIME = os.getenv('MAX_WAIT_TIME', default=60)
```

Comment:
this needs to be quick and can't wait, even 5 might be a bit long
```suggestion
MAX_MESSAGE_COUNT = os.getenv('MAX_MESSAGE_COUNT', default=100)
MAX_WAIT_TIME = os.getenv('MAX_WAIT_TIME', default=5)
```